src/settings-ui/Settings.UI.Library/PowerLauncherProperties.cs

```diff
@@ -49,6 +58,7 @@ public class PowerLauncherProperties
         public Theme Theme { get; set; }
 
         [JsonPropertyName("show_plugins_overview")]
+        [CmdConfigureIgnore]
         public int ShowPluginsOverview { get; set; }
```

Comment:
Why is this one ignored too?

Reply:
It's there on Settings, but it's an int instead of an Enum. We need to look into how to support it later on.